en/conditions.rst

```diff
@@ -3,33 +3,71 @@ Conditions
-* The free-text to describe the condition should avoid jargon
+-  When multiple conditions exist, these can be reported separately within the conditions element.
 
-2.01+ Considerations
---------------------
-In versions 2.01 and above, the following must also be considered:
+-  The conditions can be repeated in multiple languages, using separate narrative elements.
 
-* Any freetext condition text must be included in the child ``narrative`` element, which can be repeated for different languages.
+-  Three types of condition can be reported:
+
+   -  **Policy**: e.g. a particular policy needs to be implemented by the receiving organisation.
+
+   -  **Performance**: e.g. certain outputs or outcomes need to be achieved
+
+   -  **Fiduciary**: e.g. the recipient is required to use certain public financial management or public accountability measures.
+
+-  Further details about a condition can be supplied using the activity :doc:`document-link <related-documents>` element.
+
+-  If a condition relates to the whole organisation, such as organisation-wide terms and conditions, it should not be reported within an activity. Rather, it should be included as a document-link in the organisation file.
+
+Technical guidance summary
+--------------------------
+
+.. list-table::
+   :widths: 16 28 28 28
+   :header-rows: 1
+
+
+   * - Element
+     - Use
+     - Rules
+     - Guidance
+
+   * - `conditions <http://reference.iatistandard.org/activity-standard/iati-activities/iati-activity/conditions/>`__
+     - Specifies if there are terms and conditions attached to an activity.
+     - This element must occur no more than once for each activity.
+
+       The element must contain one @attached boolean. This states whether or not conditions are attached to the activity.
+
+     - When using the @attached boolean:
+
+       1 = yes
+
+       0 = no
+
+   * - `condition <http://reference.iatistandard.org/activity-standard/iati-activities/iati-activity/conditions/condition>`__
+     - Contains details of a condition.
+     - The type of condition must be provided using the `type <http://reference.iatistandard.org/codelists/ConditionType/>`__ attribute: policy, performance or fiduciary.
+     - This element can be repeated multiple times
+
+       The description of the condition is held within the narrative element.
+
+       If a condition relates to a whole organisation, it should not be reported here. Rather, it should be included as a document link in the organisation file.
+
+   * - `narrative <https://iatistandard.org/en/iati-standard/203/activity-standard/iati-activities/iati-activity/conditions/condition/narrative/>`__
+     - Short description of the condition.
+     -
+     - Further details about a condition can be supplied using the activity :doc:`document-link <related-documents>` element.
 
 .. meta::
-  :title: Conditions
+  :title: Results information
```

Comment:
The title should still say "Conditions".

Reply:
Thanks Petya, good spot